docusaurus/docs/introduction/plugin-types-usage

```diff
@@ -0,0 +1,133 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration and usage for plugin developers. General information for plugin users is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+:::
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. Make a plugin in one of several different types depending on whether you want to offer users a new way of visualizing data, a connection to a new database, or an integrated out-of-the-box experience.  
+
+The following table shows the plugin types currently supported by Grafana on the frontend and backend. You can also see whether or not each type is supported by the [Grafana API](https://grafana.com/api/plugins).
+
+|                |                 |                |                            |
+| -------------- | :-------------: | :------------: | :------------------------: |
+| **Type**       | **Frontend** |   **Backend**  | **SDK-supported** |
+| datasource     |       Yes       | Yes (optional) |             Yes            |
+| app            |       Yes       | Yes (optional) |             Yes            |
+| panel          |       Yes       |       No       |             Yes            |
+| renderer       |        No       |       Yes      |             No             |
+| secretsmanager |        No       |       Yes      |             No             |
+
+## Data source plugins
```

Comment:
Would we also like to add an extra section to each plugin type with an example on how to scaffold a new one from scratch?

Reply:
It's a good point actually to separate concept docs and tutorials. Maybe **just a link** then to make navigation and finding things easier - but I doesn't see it as something crucial.

Reply:
added a link